docs/intro/timeseries.rst

```diff
@@ -1,8 +1,8 @@
 .. _timeseries_guide:
 
-**********************
-Time series data guide
-**********************
+**********
+Timeseries
```

Comment:
Do we want to link this to the class?

Reply:
I do not, I don't like verbatim or links in titles.